PhysicsTools/PatAlgos/python/producersLayer1/patCandidates_cff.py

```diff
@@ -37,9 +37,21 @@
 
 _patCandidatesTask = patCandidatesTask.copy()
 _patCandidatesTask.remove(makePatOOTPhotonsTask)
-from Configuration.ProcessModifiers.pp_on_AA_cff import pp_on_AA
-pp_on_AA.toReplaceWith(patCandidatesTask, _patCandidatesTask) 
-pp_on_AA.toModify(patCandidateSummary.candidates, func = lambda list: list.remove(cms.InputTag("patOOTPhotons")) )
+from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
+from Configuration.Eras.Modifier_pp_on_PbPb_run3_cff import pp_on_PbPb_run3
+HI = (pp_on_AA_2018 | pp_on_PbPb_run3)
```

Comment:
this is wrong by now, after #31935 (merged Nov 3); please revert back to using `pp_on_AA`

Reply:
Done.